libraries/loader/src/loader.js

```diff
@@ -158,9 +157,17 @@ function loader(componentDirectory, virtualComponents = {}) {
       }
       return loaded[target];
     }
-
+    //method to crash the process if error occured
+    myLoader.crashOnError = function(target)
```

Comment:
This is adding the `crashError` property within the body of the `myLoader` function -- so it won't be available until after that function is called.  I think you want to define it *outside* of that function's body -- just before `return myLoader;`.

Reply:
As soon as the 
```
import loader from 'taskcluster-lib-loader'
let load = loader();
load.crashOnError(arg);//this can be called like this 
```
since we are passing as an object there wont be any problem !
if we write it as you suggested @djmitche  the scope of `load()` would not be accessible to `crashOnError()` .

Reply:
Add some tests and you'll see what I mean :)

Reply:
I got what you were trying to convey @djmitche  until we don't invoke the `load()`  method  `crashOnError()` method won't come to existence . It would be undefined until then . I should have thought about that , will update the code accordingly.

Reply:
Thank you

Reply:
Thank you